### ISSUE TITLE
If you get caught deep in a screen within a sceeen should there be easier way to get back home?

### ISSUE THREAD

**Issue by ijm8710** [lichess-org/mobile#980]
When testing opening explorer, I opened masters games off my opening input l. And then dived into further masters games on top of that.

As a result, I ended up 4-5 tiers deep of screw s within screens.

As far as I can tell, only way to then return home is to keep back swiping.

Should there be an easier way to fully go back at once by holding back arrow down for a length of time?

**Comment by veloce** [comment]
This is true only for opening explorer as far as I can tell. Otherwise you always have the option to tab the bottom tab to get back to root screen.

So I'll make this issue specific to opening explorer.

**Comment by ijm8710** [comment]
Nah, there are several other instances of this such as:

- If you go to recent game, click on opponent, then dive in on their game log and dive into one of their opponents and so on
- from broadcast: review the list of games for a player (ie magnus), then click on one of their opponents (ie fabiano), then click on one of their opponents in their game log (ie gukesh) and so on 

Rather than replacing the route, would much prefer what happens when you hold down the back arrow in safari or Wikipedia where it brings up a context menu of your recent history to jump back multiple screens at once (see photos)

![image](https://github.com/user-attachments/assets/1adf2526-f7d5-423e-8435-329d8878784f)
![image](https://github.com/user-attachments/assets/5e75252f-0207-4fab-9dfd-1bb7664a4a03)
